scripts/drive-prs.md

```
@@ -0,0 +1,353 @@
+```
+
+If a run is already `in_progress` or `queued` → skip, it is already being handled.
+
+If not, get the failure summary and delegate:
+
+```bash
+gh run view <RUN_ID> --log-failed   # get error details
+```
+
+For a **`copilot-swe-agent[bot]`** PR:
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)